usb.go

```diff
@@ -210,6 +230,38 @@ func (c *Context) OpenDevices(opener func(desc *DeviceDesc) bool) ([]*Device, er
 	return ret, reterr
 }
 
+// OpenDeviceWithFileDescriptor takes a (Unix) file descriptor of an opened USB device
+// and wraps the library around it.
+// This is particularly useful when working on Android, where the USB device must be opened
```

Comment:
perhaps "is opened" rather than "must be"? I suspect that Android NDK may offer a similar functionality, it's unlikely the "must be opened in Java SDK" is going to be true forever.

Reply:
It actually sucks a bit, but unfortunately NDK is not able to open USB devices. Yet? 
TBH I thought that the 'must be' would be a nudge for the users to realise that they will need the Java SDK to acquire permissions and open the device.
But you're correct, it's entirely possible that the NDK limitation will be lifted at some point and this docstring will become invalid.

I will change it to "can be opened"